docs/sql/statements/alter-table.rst

```diff
@@ -155,9 +155,9 @@ parameter ``number_of_shards`` can be used. For more info on that, see
 
 Can be used to add an additional column to a table. While columns can be added
 at any time, adding a new :ref:`generated column
-<sql-create-table-generated-columns>` is only possible if the table is empty.
-In addition, adding a base column with :ref:`sql-create-table-default-clause`
-is not supported. It is possible to define a ``CHECK`` constraint with the
```

Comment:
This reads like we don't support ADD COLUMN DEFAULT at all, regardless of emptiness. 

In fact, it was allowed and this PR aligns implementation and docs with generated columns and adds emptiness check and a warning for users - similar to PRIMARY KEY warning.